api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -1,25 +1,27 @@
 #!/bin/bash
 
-# This generates big query count databases cdr and public that get put in cloudsql for workbench and data browser
+# This generates big query count databases cdr that get put in cloudsql for workbench
 
 set -xeuo pipefail
 IFS=$'\n\t'
 
-
 # get options
 # --project=all-of-us-workbench-test *required
 
 # --cdr=cdr_version ... *optional
-USAGE="./generate-clousql-cdr/make-bq-data.sh --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT>"
+USAGE="./generate-clousql-cdr/make-bq-data.sh --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT> --public-project <PROJECT>"
 USAGE="$USAGE --cdr-version=YYYYMMDD"
 
 while [ $# -gt 0 ]; do
   echo "1 is $1"
+
   case "$1" in
     --bq-project) BQ_PROJECT=$2; shift 2;;
     --bq-dataset) BQ_DATASET=$2; shift 2;;
     --workbench-project) WORKBENCH_PROJECT=$2; shift 2;;
     --workbench-dataset) WORKBENCH_DATASET=$2; shift 2;;
+    --public-project) PUBLIC_PROJECT=$2; WORKBENCH_PROJECT=""; shift 2;;
```

Comment:
Why clear wb_project/wb_dataset? Should these flags be mutually exclusive?

Reply:
No and yes. This script is used in common by public and private cdr count generation as the intermediary steps are same except the rounding part in public. So the check on line 44 exits and prints usage if either workbench project & dataset / public project & dataset are not specified. The check throws unbound variable error, so i thought in case the public project is specified intializing wb project to empty variable is a good idea to deal with this error.

Reply:
Never mind. This is gone now.